index.js

```diff
@@ -28,10 +28,14 @@
   };
 
   /* istanbul ignore else */
-  if (typeof module === 'object' && typeof module.exports === 'object') {
+  if (
+    typeof module === 'object' &&
+    module != null &&
+    typeof module.exports === 'object'
+  ) {
     module.exports = mapping;
   } else {
-    self.FantasyLand = mapping;
+    globalThis.FantasyLand = mapping;
```

Comment:
Wouldn't be safer to use something like `(globalThis || self)`? Just in case...

Reply:
If `globalThis` does not exist in the environment, `globalThis || self` throws a ReferenceError. We would need to use `typeof globalThis` to first check for the presence of `globalThis`. Is this worthwhile? I'm not sure.

Reply:
Yeah, you are right. I thought short-circuiting would not throw.

Reply:
It'd be worth if we need to be backward-compatibility. But it's hard to find a reliable - and nice - way.